alacritty/src/config/window.rs

```diff
@@ -13,7 +15,7 @@ use crate::config::ui_config::{DefaultTrueBool, Delta};
 pub const DEFAULT_NAME: &str = "Alacritty";
 
 #[serde(default)]
-#[derive(Deserialize, Debug, Clone, PartialEq, Eq)]
```

Comment:
Needed to drop `Eq` as `Percentage` is effectively a `f32` and thus doesn't have `Eq`.